### ISSUE TITLE
Missed libxkbcommon include path when building in openSUSE

### ISSUE THREAD

**Issue by KottV** [surge-synthesizer/surge#1822]
**Describe the bug**
Cmake doesn't add /usr/include/libxkbcommon/ path to flags.make when building Surge from sources on openSUSE Tumbleweed

**Please let us know your surge version**
Master

**To Reproduce**
Run:

./build-linux.sh -p lv2 build

**Expected behavior**
Compile succesful

**Screenshots**
```
/home/kv/src/audio/surge/vstgui.surge/vstgui/lib/platform/linux/x11platform.cpp:25:10: fatal error: xkbcommon/xkbcommon.h: No such file or directory
   25 | #include <xkbcommon/xkbcommon.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Desktop (please complete the following information):**
openSUSE TW
Cmake 3.17.1

**Additional context**
It simple to fix by modify main CMakeLists.txt:

```
  set(OS_INCLUDE_DIRECTORIES
    src/linux
    ${CAIRO_INCLUDE_DIRS}
    ${XKBCOMMON_INCLUDE_DIRS}
    ${CMAKE_BINARY_DIR}/lintemp
    )
```
Then Surge compiles fine.
Don't know why it acts such here and works in Debian/Ubuntu.

**Comment by baconpaul** [comment]
Thanks for the super clear report. PR #1823 has a fix which i'll merge this morning. I included all the lib includes (XKB, XCB, etc...) in that PR

reason it works on ubuntu is ubuntu installs xkbcommon in /usr/include. your change (plus adding all the others for safety) is the correct one.

**Comment by baconpaul** [comment]
OK should be fixed at fd1eb2b

if it isn't please reopen this issue

also happy for you to just fire over a PR if you have a change which makes it work. Our CI will test it in ubuntu automatically for you.

Thanks again
